qa/ceph_salt/ceph_salt.yaml

```diff
@@ -0,0 +1,12 @@
+tasks:
+        - ceph_salt:
+                repo: https://github.com/ricardoasmarques/ceph-salt.git
+                branch: configurable-daemon-id-and-fsid
```

Comment:
do we really want to test this particular ceph-salt repo/branch all the time?

Reply:
We need to have PR https://github.com/ceph/ceph-salt/pull/170 merged for the test to run so I was running it on top of this branch so far

Reply:
And why it is still on ricardomarques namespace?